server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt

```diff
@@ -176,6 +192,6 @@ class KotlinLanguageServer(
     // Fixed in https://github.com/eclipse/lsp4j/commit/04b0c6112f0a94140e22b8b15bb5a90d5a0ed851
     // Causes issue in lsp 0.15
     override fun getNotebookDocumentService(): NotebookDocumentService? {
-		return null;
-	}
+        return null;
```

Comment:
Same here (though if we reformat this, we could remove the redundant semicolon and write the function as `= null` directly)